src/Components/pieChart.js

```diff
@@ -9,25 +10,77 @@ class PieChartComponent extends Component {
     super(props);
 
     this.state = {
-      labels: this.props.updatePieChartIndicators,
       shouldRedraw: false,
     };
   }
-  
-  render() {
+
+  render(props) {
```

Comment:
Is there a reason for passing in props to the render method at this point?

Reply:
@RonKbS  ,
i will rectify this